src/libPMacc/include/particles/ParticlesBase.kernel

```diff
@@ -389,11 +377,13 @@ struct KernelFillGaps
 
             if ( counterGaps == 0 )
             {
-                updateCellsForOneFrame<TileSize>(cellEntryPoint_sh, cellCount_sh, firstFrame, frameNr);
+                if (updateCellList)
+                    kernel::UpdateCellListOfFrame{}.template operator()<TileSize>(*cellEntryPoint_sh, *cellCount_sh, firstFrame, *frameNr);
```

Comment:
Did you ever try to call `operator()` with template arguments?
Hell! https://stackoverflow.com/questions/1762049/templated-operator-overload-c